packages/layout/src/components/cards/trustee/context.tsx

```diff
@@ -75,6 +75,7 @@ export interface TrusteeCardProps {
 	testId?: string | number;
 	children?: RenderProps | ReactElement;
 	cfg?: SpaceProps;
+	isRssCard?: boolean;
```

Comment:
Either make this a mandatory argument, or give it a default value, to prevent unintended effects if a client does not explicitly set it.